lib/repo/queryable.ex

```diff
@@ -74,6 +74,17 @@ defmodule ExAudit.Queryable do
     end
   end
 
+  def history_query(%{id: id, __struct__: struct}) do
+    import Ecto.Query
```

Comment:
@brunohkbx what do you think about add this `import` at the module level and only import the `from/1` function?